weblate/settings_example.py

```diff
@@ -200,6 +200,7 @@
 # GitHub username for sending pull requests.
 # Please see the documentation for more details.
 GITHUB_USERNAME = None
+GITHUB_TOKEN = None
```

Comment:
Please add this as a default value to weblate/vcs/models.py (I've moved related settings there in 7173787ec7).

Probably same is needed for GitLab...